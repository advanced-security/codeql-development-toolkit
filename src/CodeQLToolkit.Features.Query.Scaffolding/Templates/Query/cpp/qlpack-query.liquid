name: {{packName}}
version: 0.0.0
description: Default description
suites: 
license: 
dependencies:
  codeql/cpp-all: 0.3.5