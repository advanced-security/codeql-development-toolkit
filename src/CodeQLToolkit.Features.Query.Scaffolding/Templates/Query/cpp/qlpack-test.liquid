name: {{query_pack_scope}}/{{query_pack_name}}
version: 0.0.0
description: Default description
suites: 
license: 
extractor: cpp
dependencies:
  {{query_pack_dependency}}: '*'
