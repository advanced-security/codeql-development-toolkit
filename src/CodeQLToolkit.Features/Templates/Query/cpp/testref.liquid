{{query_name}}/{{query_name}}.ql