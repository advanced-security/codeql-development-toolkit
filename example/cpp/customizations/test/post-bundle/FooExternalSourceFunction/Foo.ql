import cpp
import qlt.Customizations
import semmle.code.cpp.security.FlowSources

select any(RemoteFlowSource s)