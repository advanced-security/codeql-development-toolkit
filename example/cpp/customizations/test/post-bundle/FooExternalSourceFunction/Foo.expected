| test.c:4:13:4:15 | call to foo |
